build_packages

```diff
@@ -285,6 +286,48 @@ export KBUILD_BUILD_HOST="${BUILD_HOST:-pony-truck.infra.kinvolk.io}"
 info "Merging board packages now"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" "$@"
 
+info "Merging sysext packages now"
+for sysext in "${EXTRA_SYSEXTS[@]}"; do
+  IFS="|" read SYSEXT_NAME PACKAGE_ATOMS USEFLAGS ARCHES < <(echo "$sysext");
+
+  arch_array=(${ARCHES//,/ })
+  if [[ ! -z "$ARCHES" ]]; then
+    should_skip=1
+    for arch in "${arch_array[@]}"; do
+      if [ "$arch" == "$ARCH" ]; then
```

Comment:
Please be more consistent with shell standards. If you know this is Bash, rather than POSIX shell (/bin/sh) then always use `[[` over `[` because it's safer and doesn't require quoting the LHS. I'm flagging this particular example because `==` is a Bash extension, so you would need to use `=` to support POSIX shells. shellcheck doesn't flag it but mixing `[` with `==` is confusing.

```suggestion
      if [[ $arch == "$ARCH" ]]; then
```